ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/internal_vlan_order.schema.yml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  internal_vlan_order:
+    documentation_options:
+      filename: Fabric Variables
+      table: VLAN Settings
+    type: dict
+    description: Internal vlan allocation order and range.
+    required: true
+    keys:
+      allocation:
+        type: str
+        valid_values:
+        - ascending
+        - descending
+        default: ascending
+      range:
+        type: dict
+        keys:
+          beginning:
+            type: int
+            convert_types:
+            - str
+            description: VLAN ID
+            default: 1006
+            min: 1
+            max: 4094
+          ending:
+            type: int
+            convert_types:
+            - str
+            description: VLAN ID
+            default: 1199
+            min: 1
+            max: 4094
```

Comment:
so I know it sounds dumb but maybe let's add that beginning must be lower than ending.

Reply:
and maybe we can mention the min and max ranges in the description somewhere? Preferably on Line27 and Line35?

Reply:
@sugetha24 the min, max and default are shown documentation table.

Reply:
addressed in [76813b1](https://github.com/aristanetworks/ansible-avd/pull/2510/commits/76813b11d3b1383ef83bfea0e5c7781a986a2a8a)